packages/rspack/src/compiler.ts

```diff
@@ -284,10 +285,15 @@ class Compiler {
 				});
 		};
 		const options = getRawOptions(this.options, this, processResource);
-
+		const instanceBindingVersion =
+			require("@rspack/binding/package.json").version;
+		bindingVersionCheck(instanceBindingVersion, err => {
+			if (err) throw err;
```

Comment:
This part should follow the callback rule too. You can change this getter to `#getInstance(callback)`

Reply:
OK，I have change it